src/components/NightModeToggle/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component, PropTypes } from 'react';
+
+import SwitchToggle from 'components/SwitchToggle';
+
+export default class NightModeToggle extends Component {
+  static propTypes = {
+    isNightMode: PropTypes.bool.isRequired,
+    onToggle: PropTypes.func.isRequired
+  };
+
+  static defaultProps = {
+    isNightMode: false
+  };
+
+  toggleNightMode = () => {
+    const { isNightMode } = this.props;
+
+    if(isNightMode)
+      document.body.classList.add('night-mode');
```

Comment:
I always thought `toggle` is a jquery function, thanks for good suggestion. Also brackets aren't required for single statement, but if we are following some conventions happy to follow them.